app/models/custom_button_event.rb

```diff
@@ -1,2 +1,12 @@
 class CustomButtonEvent < EventStream
+  virtual_column :button_name, :type => :string
+  virtual_column :automate_entry_point, :type => :string
+
+  def automate_entry_point
+    full_data[:automate_entry_point]
+  end
+
+  def button_name
+    full_data[:button_name]
```

Comment:
@lfu I would suggest capturing the button ID as well.  Then this method should use the current button name, incase it gets renamed, and fall back to the name of the button when it was run if the button get deleted.